### ISSUE TITLE
Simple Grade Entry: Create unit tests and functional tests for CSV upload/download

### ISSUE THREAD

**Issue by benjaminvialle** [MarkUsProject/Markus#99]
Unit tests and functional tests need to be created for uploading/downloading the grades table as a CSV file. (see review request#497)

**Comment by MatakuT** [comment]
made a progress on this issue: #2077

**Comment by david-yz-liu** [comment]
@MatakuT You could also write tests in which correct files are uploaded, and make sure all the marks are there.

**Comment by MatakuT** [comment]
Will work on that

**Comment by MatakuT** [comment]
made a progress: #2092 
There seem to be a conflicting pull request. Does it conflict with the previous fix: #2077?
This conflict does not allow Travis CI to do run the testing.

**Comment by david-yz-liu** [comment]
@MatakuT You should pull from master and resolve the conflict yourself.

**Comment by MatakuT** [comment]
@david-yz-liu I tried the following to find out what is conflicting (I assume it's grade_entry_form_spec.rb file)
Switched to branch 'iss99_additional_tests'
moskalem:Markus alzu$ git merge master
Already up-to-date.
moskalem:Markus alzu$ git merge origin/master
Already up-to-date.

I have never resolved a merge conflict before, so not quite sure what to do next. Found few suggestions to force merge/rebase into remote master, but I don't know git too well and too afraid to screw things up.

**Comment by david-yz-liu** [comment]
You need to be pulling from upstream master, not origin/master. (Origin is you, upstream is MarkUsProject.)

After that, you can consult this: https://help.github.com/articles/resolving-a-merge-conflict-from-the-command-line/

**Comment by MatakuT** [comment]
After spending 4 hours and not being able to resolve the conflict I created a new branch. This one 100% in synch with MarkUsProject master branch. #2092 has been closed and iss99_additional_tests branch was deleted. I still have a local copy just in case.
I realize that it is not a good practice, but I just got into a dead-end and decided to take an easy approach. I learned my lesson with why it was conflicting in the first place and will try to not repeat similar mistake again.
The new PR request can be found here: #2098
